azurepipelines/adu-debian-arm32-build.yml

```diff
@@ -28,7 +28,7 @@ trigger:
   branches:
     include:
       - main
```

Comment:
This pipeline will still trigger on check-ins to main ( as well as pr's opened against main, which for adu agent is only when dev branches are merged back in correct?) - intended?